app/config/api_filters.yml

```diff
@@ -50,6 +50,11 @@ services:
               url: 'exact'
         tags: [{ name: 'api_platform.filter', id: 'event.search' }]
 
+    resource.event.exists_filter:
```

Comment:
I find `…?exists[feed]=true` hard to understand, but that's not important in this project at this point. However, you could consider renaming `exists` to `has` (cf. https://api-platform.com/docs/v2.7/core/filters/#using-a-custom-exists-query-parameter-name).

Reply:
We are still on `v2.2`. Doesn't seem like that option is available: https://api-platform.com/docs/v2.2/core/filters/#exists-filter